paimon-spark/paimon-spark-common/src/main/scala/org/apache/paimon/spark/commands/SchemaHelper.scala

```diff
@@ -68,6 +68,27 @@ private[spark] trait SchemaHelper extends WithFileStoreTable {
     }
   }
 
+  def mergeV2Schema(rowSchema: StructType, options: Options): StructType = {
```

Comment:
Let rename this method `mergeSchema`, and make the original `mergeSchema(sparkSession: SparkSession, input: DataFrame, options: Options))` to call this.

Reply:
done